packages/wasm/crate/src/planner.rs

```diff
@@ -410,19 +410,27 @@ pub async fn plan_transaction(
         let mut nonce = [0u8; 32];
         OsRng.fill_bytes(&mut nonce);
 
+        let description = DutchAuctionDescription {
+            start_height: description.start_height,
+            end_height: description.end_height,
+            step_count: description.step_count,
+            input,
+            output_id,
+            min_output,
+            max_output,
+            nonce,
+        };
+
+        save_auction_nft_metadata_if_needed(
+            &description,
+            &storage,
+            // When scheduling a Dutch auction, the sequence number is always 0
+            0,
+        )
+        .await?;
```

Comment:
Now that we're actually able to render a `ValueView` of the action NFT in the transaction approval dialog, we need to optimistically save the NFT metadata to the database so that the dialog has metadata to render.